components/pkg-export-kubernetes/src/manifestjson.rs

```diff
@@ -71,30 +68,25 @@ impl ManifestJson {
             binds: binds,
         }
     }
+}
 
+impl Into<String> for ManifestJson {
     /// Convert into a string. The returned string is the final manifest YAML file content, ready
     /// for consumption by a Kubernetes cluster.
-    ///
-    /// # Errors
-    ///
-    /// * Rendering from the template fails. This can only happen if the template has a syntax
-    /// error or the command-line arguments are incorrectly formatted. In both cases, it's a
-    /// most likely a programmer error if it happens.
-    // TODO: Implement TryInto trait instead when it's in stable std crate
-    pub fn into_string(&self) -> Result<String> {
+    fn into(self) -> String {
         let r = Handlebars::new()
             .template_render(MANIFESTFILE, &self.main)
-            .map_err(SyncFailure::new)?;
+            .expect("Rendering of manifest from template failed");
```

Comment:
Maybe add a gist of your commit message here as a comment? Just as a general rule to document the `unwrap` or `expect` calls.